src/bindings/python/src/openvino/frontend/tensorflow/node_decoder.py

```diff
@@ -26,25 +26,33 @@ def tf_type_to_ov_type(tf_type_int):
 
 def tf_attr_to_numpy(attr):
     attr_type = attr.WhichOneof("value")
+    # described in https://www.tensorflow.org/api_docs/python/tf/compat/v1/AttrValue
     if attr_type == "func":
         return attr.func.name
-    if attr_type == "s":
+    elif attr_type == "s":
         try:
             return attr.s.decode("utf-8")
         except UnicodeDecodeError:
             return attr.s
-    if attr_type == "f":
+    elif attr_type == "f":
         return np.float32(attr.f)
-    if attr_type == "type":
+    elif attr_type == "type":
         return tf_type_to_ov_type(attr.type)
-    if attr_type == "list":
+    elif attr_type == "list":
         list_value = attr.list
         fields = list_value.ListFields()
         if fields and len(fields) > 0 and len(fields[0]) > 1:
             return list(fields[0][1])
         else:
             return None
-    if attr_type is None:
+    elif attr_type == "shape":
```

Comment:
Can we delete the branch checking `name == "shape" or name == "_output_shapes"` (lines 147-158)?

Reply:
it is better to leave such refactoring for the next release and to have it not risky, now I just cover remained cases.

Reply:
Also, added check if variable is `None`